src/masternode-budget.cpp

```diff
@@ -1961,7 +1961,6 @@ std::string CFinalizedBudget::GetStatus() const
 
 void CFinalizedBudget::SyncVotes(CNode* pfrom, bool fPartial, int& nInvCount) const
 {
-    LOCK(cs);
```

Comment:
would be nice to have an AssertLockHeld or a comment making more visible why we don't need to guard this methods. Would be less error pruning for the future for anyone who touches this code.

-- Same applies to the other methods that have the locks removed and, in a first glance without checking the caller, looks like need to be protected --

Reply:
Wouldn't add `AssertLockHeld` everywhere, as that's not how other locks are treated elsewhere in the code. We'll use thread-safe-analysis annotation in the future.

Anyway, mapVotes (here) is guarded by `cs_votes` which is introduced in a subsequent commit.